app/assets/stylesheets/tiny_mce.scss

```diff
@@ -66,4 +66,32 @@
 .mce-toolbar {
   background: $color-white !important;
 }
+
+.tinymce-cancel-notification-button {
+  cursor: pointer;
+
+  .fas {
+    color: $color-silver-chalice;
+    font-family: 'Font Awesome 5 Free';
+    font-weight: 501;
+    margin-left: 10px;
+    margin-top: 4px;
+  }
+}
+
+.restore-draft-notification {
+  background: $state-info-bg !important;
+  height: 25px !important;
+  padding-left: 10px !important;
+  padding-right: 10px !important;
+  padding-top: 5px !important;
+
```

Comment:
Can you make it without `!important`

Reply:
The problem is that TinyMCE spreadsheet sets style for every descendant within the editor, so the toolbar is also affected and I am not sure how to force it without `!important`.

Reply:
Ok, it's very long to write how to make it. I will show when you will be in office next time. At least can you combine padding in one line: `padding: 5px 10px !important;`